app/js/controllers/foyer/individuForm.js

```diff
@@ -132,6 +128,14 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
         return false;
     };
 
+    $scope.captureSejour10ans = function() {
+        return individu.nationalite == 'autre';
+    };
+
+    $scope.captureSejour5ans = function() {
+        return individu.nationalite == 'autre' && angular.isDefined(individu.titreSejour10ans) && (! individu.titreSejour10ans);
```

Comment:
Isn't all this equivalent to `individu.titreSejour10ans === false`? Since it would be `undefined` unless captured…